README.md

```diff
@@ -21,6 +27,20 @@ To install the `irma` command line tool:
 
     go install ./irma
 
+You can also include the `irma` command line tool in a Docker image, using a base image of your choice. The default base image is Debian's `buster-slim`.
+
+    docker build --build-arg BASE_IMAGE=alpine -t privacybydesign/irma:edge .
+
+When you build for production, we recommend you to build the [latest release](/releases/latest). You should replace `v0.0.0` with the latest version number.
+
+    docker build -t privacybydesign/irma https://github.com/privacybydesign/irmago#v0.0.0
```

Comment:
It is a bit of a pity that there is no default `latest` tag or another way to choose the latest release.